README.md

```diff
@@ -96,9 +56,8 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
```

Comment:
- [OPTIONAL] I think it's better to remove this line with guidelines for README use. The text after the `>` at the beginning of each session is a guide. Your README will look more professional without these guidelines. If you decide to remove these guidelines, remove them from all sessions.

Reply:
I will remove all those texts. Thanks.